kokoro/ubuntu/continuous.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+#
+# This is just an initial version based on the Travis build script, the build
+# needs UI, which haven't been tested, just copied over from .travis.yml
```

Comment:
xvfb is not listed among the installed packages and I think I ran the script and saw it failing. I haven'r tested the DISPLAY redirection.